demos/common/cpp/utils/include/utils/common.hpp

```diff
@@ -54,6 +54,12 @@ inline slog::LogStream& operator<<(slog::LogStream& os, const InferenceEngine::V
     return os;
 }
 
+inline slog::LogStream& operator<<(slog::LogStream& os, const ov::Version& version) {
```

Comment:
this function not needed any more, because of https://github.com/openvinotoolkit/openvino/pull/8340/files#diff-5474cb1155accc574f3fcc160d216c1a468a6aa9751ef6f902b05bb8d1afd234R13-R21

Reply:
It's overloaded for our logger, for pretty printing with [INFO] on each line, so it's needed actually.